apps/vite/src/components/MonthCalendar.tsx

```
@@ -130,14 +130,12 @@ export default function MonthCalendar({
                     style={{
                       backgroundColor: color ? colors[color] : "#475569",
                     }}
-                    className={classNames(
-                      `mx-[1px] inline-block h-1 w-1 rounded-full`
-                    )}
+                    className={cn(`mx-[1px] inline-block h-1 w-1 rounded-full`)}
```

Biome

[error] Do not use template literals if interpolation and special-character handling are not needed.

Unsafe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)